code/processes/dqc.q

```diff
@@ -219,10 +224,13 @@ reruncheck:{[chkid]
   .timer.removefunc'[exec funcparam from .timer.timer where `.dqe.writedown in' funcparam];                     /- clear writedown timer
   .timer.removefunc'[exec funcparam from .timer.timer where `.dqe.writedownconfig in' funcparam];               /- clear writedownconfig timer
   delete configtable from `.dqe;
-  .dqe.init[];
   .dqe.currentpartition:pt+1;
+  if[(`timestamp$.dqe.currentpartition)>=.eodtime.nextroll;.eodtime.getroll[`timestamp$.dqe.currentpartition];.lg.o[`dqc;"Manually update .eodtime.nextroll as it was incorrect"]]
```

Comment:
Comment is a bit clunky. Needs to say about moving `nextroll` to the next day. Something like "Moving nextroll to match current partition".